deis-logger-beta1-test/manifests/deis-logger-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+    app: deis-logger
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: http
+  - port: 514
+    name: transport
+    targetPort: 1514
+    protocol: UDP
+  selector:
+    app: deis-logger
```

Comment:
:+1: to including newlines at the ends of files by default. Some tools will complain about the lack of a final newline, but none will complain about it being there.